tensorflow_addons/metrics/cohens_kappa_test.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for Cohen's Kappa Metric."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+
+from tensorflow_addons.metrics import CohensKappa
+from tensorflow_addons.utils import test_utils
+
+@test_utils.run_all_in_graph_and_eager_modes
+class CohensKappaTest(tf.test.TestCase):
+  def test_config(self):
+    kp_obj = CohensKappa(name='cohens_kappa')
+    self.assertEqual(kp_obj.name, 'cohens_kappa')
+
+  def test_kappa(self):
```

Comment:
You can split `test_kappa` into three separate test cases for random score, perfect score and worse than random. Also use parameterized tests for `sample_weight` like [here](https://github.com/tensorflow/tensorflow/blob/186f573d7067b1df32e3bf2ffb7d34cffcdb4c64/tensorflow/python/data/experimental/kernel_tests/take_while_test.py#L53).

Reply:
I have separated the tests into three different test cases. Is the parameterized thing really required? I don't think so as there are many test cases where it is not used in the `tf.keras` core

Reply:
Nope, not required. Just a suggestion.